### ISSUE TITLE
Inventory System Connector bug when used with a Supplier Pipe

### ISSUE THREAD

**Issue by Nex4rius** [RS485/LogisticsPipes#762]
https://www.youtube.com/watch?v=dDNbJlSpKK4&feature=youtu.be

logisticspipes-0.9.2.19
Infitech2 modpack v3.1.5

**Comment by davboecki** [comment]
What are those yellow blocks?
Could you try to simply use a double chest?

**Comment by Nex4rius** [comment]
The yellow blocks are "Large Electrum Item Pipe" from gregtech (instantanous transport from one end to another to remove waiting time on long distances)
gregtech_1.7.10-5.08.25

I tested it now with a double chest and the ISC. It puts the item into the chest but the other ISC doesnt pull it out (request with "Request Logistics Pipe Mk2" still works).
edit: Sometimes it also gets stuck and cannot insert into the chest.

**Comment by hron84** [comment]
@DarknessShadow and you removed GT pipes completely from your system? I think LP currently does not have a support for these pipes, I recommend you to use Thermal Dynamics warp pipes instead. They require power, but LP currently have an integration with TD.

**Comment by Nex4rius** [comment]
@hron84 
Yes I removed the GT pipes completely but it also happens WITHOUT the gregtech pipes so its not a problem with gregtech.
Btw I'm playing on a hardmode gregtech modpack (Infitech2) that doesn't have Thermal Dynamics.

**Comment by hron84** [comment]
@DarknessShadow OK, sry, it was not clear from your previous comment, that's why i asked that.
